src/main/resources/persistence.properties

```diff
@@ -4,7 +4,8 @@ hibernate.archive.autodetection = class
 
 javax.persistence.jdbc.url = jdbc:postgresql://188.166.92.29:5432/moodcat_db
 javax.persistence.jdbc.user = moodcat
-hibernate.hbm2ddl.auto = create-drop
+#hibernate.hbm2ddl.auto = create-drop
+hibernate.hbm2ddl.auto = update
```

Comment:
I think rebasing and squashing is a good idea, to keep changes simple. I don't think it is too harmful for the history as it is mostly a seperate feature we developed.